zone.js

```diff
@@ -8,7 +8,7 @@ var zone = null;
 function Zone(parentZone, data) {
   var zone = (arguments.length) ? Object.create(parentZone) : this;
 
-  zone.parent = parentZone;
+  zone.parent = parentZone || null;
```

Comment:
right. that's what this PR is doing and that's an intentional change :) see a few lines below.